.github/workflows/release-channels.yml

```diff
@@ -0,0 +1,52 @@
+name: set ZITADEL stable image
```

Comment:
```suggestion
name: ZITADEL Release tags
```

Since it might propagate multiple channels if feel it should be agnostic

Reply:
fine with me